.github/workflows/linux.yml

```diff
@@ -971,29 +985,39 @@ jobs:
 
       - name: Extract OpenVINO packages
         run: |
-          pushd ${{ env.INSTALL_DIR }}
-            tar -xzf openvino_package.tar.gz -C ${{ env.INSTALL_DIR }} && rm openvino_package.tar.gz || exit 1
+          pushd ${INSTALL_DIR}
+            tar -xzf openvino_package.tar.gz -C ${INSTALL_DIR} && rm openvino_package.tar.gz || exit 1
           popd
-          pushd ${{ env.INSTALL_TEST_DIR }}
-            tar -xzf openvino_tests.tar.gz -C ${{ env.INSTALL_DIR }} && rm openvino_tests.tar.gz || exit 1
+          pushd ${INSTALL_TEST_DIR}
+            tar -xzf openvino_tests.tar.gz -C ${INSTALL_DIR} && rm openvino_tests.tar.gz || exit 1
           popd
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
       - name: Install Python wheels
         run: |
-          python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
+          python3 -m pip install --upgrade pip
+          python3 -m pip install openvino --find-links=${INSTALL_DIR}/tools
```

Comment:
BTW, `find-links` does not guarantee installation from this folder. Safer to use full path to install wheel package

Reply:
See how it's done for 2022.3 branch https://github.com/openvinotoolkit/openvino/pull/19031

Reply:
it may cause other issues (cache ignoring), changed, testing that

Reply:
cache works fine